src/lib/one-graph/cli-netlify-graph.js

```diff
@@ -241,6 +241,31 @@ const ensureFunctionsPath = (netlifyGraphConfig) => {
   fs.mkdirSync(fullPath, { recursive: true })
 }
 
+let disablePrettierDueToPreviousError = false
+
+const runPrettier = async (filePath) => {
+  if (disablePrettierDueToPreviousError) {
+    return
+  }
+
+  const command = `prettier --write ${filePath}`
+  try {
+    const commandProcess = execa.command(command, {
```

Comment:
Small note: if `filePath` has a space in its path (which is common on Windows, e.g. `C:\Program Files`), it will be interpreted as two different arguments by `execa.command()`.
Instead, using `execa('prettier', ['--write', filePath])` will prevent this problem.